src/Data/Fits.hs

```diff
@@ -308,57 +241,118 @@ parsePix c bpf bs = return $ runGet (getPixs c bpf) bs
 {- `pixDimsByCol` takes a list of Axis and gives a column-row major list of
     axes dimensions.
 -}
-pixDimsByCol :: [Axis] -> [Int]
-pixDimsByCol = map axisElementCount
+pixDimsByCol :: Axes -> [Natural]
```

Comment:
I'm not sure that using `Natural` here is a good idea. I'm a little bit behind on understanding the runtime representation, but the choice of `[Int]` was not because I didn't want to use fancier number types, but due to possible issues getting `[Natural]` to coalesce for downstream users wanting to do things like using Accelerate or other DataFrame like frameworks. Now I could be persuaded that this is a non-issue, but I've shied away from the `Natural` number type for ergonomic reasons for downstream users.

What do you think? Am I just being a stick in the mud here?

Reply:
I don't have a strong opinion, nor experience with the mentioned frameworks yet. I agree, let's use whatever is most ergonomic downstream. I'll strip out all Natural uses and push the changes